src/index.html

```diff
@@ -5,4 +5,151 @@
 order: 1
 ---
 
-<p>Add the HTML for your homepage here.</p>
+<iframe src="https://www.youtube.com/embed/z_jefhcVRAk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="about">
```

Comment:
In general, at this stage where you are establishing the markup but don't yet know what you'll need to do for styles, I would not include class names yet. Add classes as they are needed in your CSS, but only as needed.